dev-docs/adding-data/add_scanjobs.go

```diff
@@ -139,7 +141,9 @@ func main() {
 	})
 	ec2NodeID2, err := gatewayNodesClient.Create(suite.ctx, &gwnodes.Node{
 		Name: "inspec-target-rhel7-acceptance.cd.chef.co",
-		Tags: []*query.Kv{},
+		Tags: []*query.Kv{
```

Comment:
this is so the `load_scan_jobs` command will add some nodes with env values by default